wp-content/themes/genesis-starter/gulp/config.js

```diff
@@ -73,7 +73,7 @@ export default {
       ]
     }
   },
-  svgSpriteSrc: 'svg-sprite/**/*.svg',
+  svgSpriteSrc: '**/svg-sprite/**/*.svg',
```

Comment:
Before didn't work. Now it works. And no, it doesn't include any svg files that aren't in the svg-sprite folder.

If anyone else wants to test and come up with another solution, feel free to do so.